antarest/launcher/service.py

```diff
@@ -281,10 +305,11 @@ def _filter_from_user_permission(
                 ):
                     allowed_job_results.append(job_result)
             except StudyNotFoundError:
-                logger.info(
-                    f"Removing job result {job_result.id} because of missing study"
-                )
-                self.job_result_repository.delete(job_result.id)
```

Comment:
Why not keep it in a else statement ?

Reply:
It's intended, because if the user is not admin, it will remove the job result. But we don't want that. Admin should be able to view all job results and decide to delete them manually.